edgelight/edgelight.ino

```diff
@@ -38,218 +36,226 @@ SOFTWARE.
 
 // setting this will allow a GPIO held on boot to reset the wifi credentials and file system
 #define GPIO_RESET_CREDENTIALS -1
 
 // set the board type here.  check the pin definitions below to match your actual board
+// #define CAMERA_MODEL_ESP32_CAM_BOARD 1
 #define CAMERA_MODEL_M5STACK_PSRAM 1
 
 #if defined(CAMERA_MODEL_ESP32_CAM_BOARD)
-#define PWDN_GPIO_NUM     32
-#define RESET_GPIO_NUM    -1
-#define XCLK_GPIO_NUM      0
-#define SIOD_GPIO_NUM     26
-#define SIOC_GPIO_NUM     27
-
-#define Y9_GPIO_NUM       35
-#define Y8_GPIO_NUM       34
-#define Y7_GPIO_NUM       39
-#define Y6_GPIO_NUM       36
-#define Y5_GPIO_NUM       21
-#define Y4_GPIO_NUM       19
-#define Y3_GPIO_NUM       18
-#define Y2_GPIO_NUM        5
-#define VSYNC_GPIO_NUM    25
-#define HREF_GPIO_NUM     23
-#define PCLK_GPIO_NUM     22
-
-#define GPIO_LED_FLASH    4
+#define PWDN_GPIO_NUM 32
+#define RESET_GPIO_NUM -1
+#define XCLK_GPIO_NUM 0
+#define SIOD_GPIO_NUM 26
+#define SIOC_GPIO_NUM 27
+
+#define Y9_GPIO_NUM 35
+#define Y8_GPIO_NUM 34
+#define Y7_GPIO_NUM 39
+#define Y6_GPIO_NUM 36
+#define Y5_GPIO_NUM 21
+#define Y4_GPIO_NUM 19
+#define Y3_GPIO_NUM 18
+#define Y2_GPIO_NUM 5
+#define VSYNC_GPIO_NUM 25
+#define HREF_GPIO_NUM 23
+#define PCLK_GPIO_NUM 22
+
+#define GPIO_LED_FLASH 4
 
 #elif defined(CAMERA_MODEL_M5STACK_PSRAM)
-
 #define GPIO_RESET_CREDENTIALS 38
 
-#define PWDN_GPIO_NUM     -1
-#define RESET_GPIO_NUM    15
-#define XCLK_GPIO_NUM     27
-#define SIOD_GPIO_NUM     25
-#define SIOC_GPIO_NUM     23
-
-#define Y9_GPIO_NUM       19
-#define Y8_GPIO_NUM       36
-#define Y7_GPIO_NUM       18
-#define Y6_GPIO_NUM       39
-#define Y5_GPIO_NUM        5
-#define Y4_GPIO_NUM       34
-#define Y3_GPIO_NUM       35
-#define Y2_GPIO_NUM       32
-#define VSYNC_GPIO_NUM    22
-#define HREF_GPIO_NUM     26
-#define PCLK_GPIO_NUM     21
+#define PWDN_GPIO_NUM -1
+#define RESET_GPIO_NUM 15
+#define XCLK_GPIO_NUM 27
+#define SIOD_GPIO_NUM 25
+#define SIOC_GPIO_NUM 23
+
+#define Y9_GPIO_NUM 19
+#define Y8_GPIO_NUM 36
+#define Y7_GPIO_NUM 18
+#define Y6_GPIO_NUM 39
+#define Y5_GPIO_NUM 5
+#define Y4_GPIO_NUM 34
+#define Y3_GPIO_NUM 35
+#define Y2_GPIO_NUM 32
+#define VSYNC_GPIO_NUM 22
+#define HREF_GPIO_NUM 26
+#define PCLK_GPIO_NUM 21
 #endif
 
-
-char groundlight_endpoint[40]="api.groundlight.ai";
+// Options to configure these settings will be surfaced when the user connects to the device's AP
+char groundlight_endpoint[40] = "api.groundlight.ai";
 char groundlight_API_key[75] = "api_yourgroundlightapikeyhere";
 char groundlight_det_name[100] = "det_xxxyourdetectoridhere";
 char groundlight_det_query[100] = "your query text here?";
 char groundlight_det_confidence[5] = "0.9";
 char groundlight_take_action_on[6] = "YES";
 char groundlight_action_channel[10] = "SLACK";
 char slack_url[100] = "https://hooks.slack.com/services/blah/blah/blah";
-char delay_between_queries[10] = "30000";
+char delay_between_queries_ms[10] = "30000";
```

Comment:
pull from the config default?

Reply:
Decided the default config was a bit confusing. I just treat the values here as the defaults now.